src/Aspire.Hosting/Extensions/ParameterResourceBuilderExtensions.cs

```diff
@@ -37,6 +38,15 @@ public static IResourceBuilder<ParameterResource> AddParameter(this IDistributed
     {
         var resource = new ParameterResource(name, callback, secret);
         return builder.AddResource(resource)
+                      .WithDashboardState(state => state with
+                      {
+                          ResourceType = "Parameter",
+                          Properties = [
+                            ("Value", callback()),
```

Comment:
This could be problematic, because we don't want this to throw because the value is missing. These callbacks are evaluated out of order with when DCP starts.